### ISSUE TITLE
Init scripts also on initialClientRender

### ISSUE THREAD

**Issue by raae** [raae/gatsby-remark-oembed#74]
@adamwiggall made me aware we need to `processInstagram` and `loadTwitter` also on `onInitalClientLoad`.

I would combine `processInstagram` and `loadTwitter` in a `initScripts` function and call that both from both `onInitalClientLoad` and `onRouteUpdate`. 

The code for this can be found in `gatsby-browser.js`.

**Comment by victornguyen** [comment]
In working on this, I've noticed that `onRouteUpdate()` gets [called on the initial page load](https://github.com/gatsbyjs/gatsby/pull/1320) anyway?

I've PR'ed up the change anyway in case making use of `onInitialClientRender()` solves something else I'm not aware of. The option is yours 😄

**Comment by raae** [comment]
I created a contentful project using their starter to test something else. And there I found that `onRouteUpdate()` and `onInitialClientRender()` is called before the page templates `render`. 

I think this kind of issue prompted this proposed solution. But it does not fix it as both are called prematurely. I cannot figure out why this is the case and what is causing it. A hack is to let  `onInitialClientRender()` call `initScript()` after a timeout, but that is very hacky.
